src/main/webapp/app/entities/patient/patient-details.component.ts

```diff
@@ -2,16 +2,30 @@ import { defineComponent, inject, ref, type Ref } from 'vue';
 import { useI18n } from 'vue-i18n';
 import { useRoute, useRouter } from 'vue-router';
 
+import { Line } from 'vue-chartjs';
+import { BarElement, CategoryScale, Chart as ChartJS, Legend, LinearScale, LineElement, PointElement, Title, Tooltip } from 'chart.js';
 import PatientService from './patient.service';
+import MesureService from '../mesure/mesure.service';
 import useDataUtils from '@/shared/data/data-utils.service';
 import { type IPatient } from '@/shared/model/patient.model';
 import { useAlertService } from '@/shared/alert/alert.service';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faArrowsUpDown, faCakeCandles, faDoorOpen, faGenderless, faLocationDot } from '@fortawesome/free-solid-svg-icons';
+
+ChartJS.register(Title, Tooltip, Legend, BarElement, CategoryScale, LinearScale, PointElement, LineElement);
 
 export default defineComponent({
   compatConfig: { MODE: 3 },
   name: 'PatientDetails',
+  components: { Line },
   setup() {
+    library.add(faLocationDot);
```

Comment:
might be factorized like this : 
const iconsToAdd = [faLocationDot, faCakeCandles, faGenderless, faArrowsUpDown, faDoorOpen];
    library.add(...iconsToAdd);